SharedSources/MovieCollectionViewCell.swift

```diff
@@ -0,0 +1,64 @@
+/*****************************************************************************
+ * MovieCollectionViewCell.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Carola Nitz <nitz.carola # googlemail.com>
+ *
+ * Refer to the COPYING file of the official project for license.
+*****************************************************************************/
+
+import Foundation
+
+class MovieCollectionViewCell: MediaCollectionViewCell {
+
+    @IBOutlet weak var thumbnailView: UIImageView!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var newLabel: UILabel!
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var progressView: UIProgressView!
+
+    override class var nibName: String {
+        return "MovieCollectionViewCell"
```

Comment:
What about using `String(describing:)` here?

Reply:
could use it yes